entries/b/bill.com.json

```diff
@@ -1,7 +1,6 @@
 {
   "Bill": {
     "domain": "bill.com",
-    "url": "https://www.bill.com",
```

Comment:
This one too

Reply:
Got that one now. Thanks for keeping me honest. Got distracted while working on this.